tests/MonitoringFunctions.Test/TestHelperMethods.cs

```diff
@@ -34,5 +36,43 @@ public async Task TestCheckUrlAccessFailure(string url)
                 // Test passed
             }
         }
+
+        [DataRow("-DryRun -c 3.0")]
+        [DataRow("-DryRun -c release/5.0.1xx-preview7")]
+        [DataRow("-DryRun -Version LTS")]
+        [TestMethod]
+        public async Task TestExecuteInstallScriptPs1(string cmdArgs = "-DryRun")
```

Comment:
consider async suffix

Reply:
Isn't asking for Async suffix on test methods is the kind of bureaucracy that we are trying to avoid?

Reply:
if there is a best practice, why make an exception for test code?  It's like you are purposely not trying to do your best in test code :)

Reply:
this is minor, up to you to resolve as is (or do the right thing for the code, the team, the company and mankind)